webapp/components/ContributionForm/ContributionForm.spec.js

```diff
@@ -384,6 +384,15 @@ describe('ContributionForm.vue', () => {
           await wrapper.find('form').trigger('submit')
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
+
+        it('supports deleting a teaser image', async () => {
+          expectedParams.variables.image = null
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-18T16:25:46Z" title="Tuesday, February 18th 2020, 5:25:46 pm +01:00">Feb 18, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -384,6 +392,15 @@ describe('ContributionForm.vue', () => {
           await wrapper.find('form').trigger('submit')
           expect(mocks.$apollo.mutate).toHaveBeenCalledWith(expect.objectContaining(expectedParams))
         })
+
+        it('deletes a teaser image', async () => {
+          expectedParams.variables.image = null
```


Could you please hard-code the entire `expectedParams` here?

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-02-18T18:35:55Z" title="Tuesday, February 18th 2020, 7:35:55 pm +01:00">Feb 18, 2020</time>_

---


as we discussed in our pairing session, I understand the motive for this, but `expectedParams` is quite big, and is only 10 lines of code above this, so I think it's ok like this.